spec/views/users/show.html.erb_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page', type: :feature do
```

Comment:
-  [ ] There is a typo here. Instead of "User index page', it should be 'User show page'.

Reply:
Done